client/src/components/pages/common/Login/index.js

```diff
@@ -41,127 +32,101 @@ class Login extends React.Component {
                 email,
                 password,
               })
-              .then((response) => {
-                this.setState({ role: response.data.role }, () => {
-                  this.redirect();
-                });
+              .then(({ data: { role } }) => {
+                if (role === 'admin') {
+                  window.location.replace('/dashboard');
+                } else {
+                  window.location.replace('/steps');
+                }
               })
-              .catch(() =>
-                this.setState({
-                  message: 'Please Check your Email or Password',
-                })
-              )
-          : this.setState({ message: 'Please Check your Email or Password' })
+              .catch(() => setMessage(['Please Check your Email or Password']))
+          : setMessage(['Please Check your Email or Password'])
       )
-      .catch(({ errors }) => this.throwMessage(errors));
+      .catch(({ errors }) => throwMessage(errors));
   };
 
-  updateEmail = (e) => {
-    this.setState({ email: e.target.value });
+  const updateEmail = (e) => {
+    setEmail(e.target.value);
   };
 
-  updatePassword = (e) => {
-    this.setState({ password: e.target.value });
+  const updatePassword = (e) => {
+    setPassword(e.target.value);
   };
 
-  redirect() {
-    const { role } = this.state;
-    const { history } = this.props;
-    if (role === 'applicant') {
-      history.push('/steps');
-    } else if (role === 'admin') {
-      history.push('/dashboard');
-    }
-  }
-
-  render() {
-    const { message } = this.state;
-    return (
-      <ThemeProvider theme={Theme}>
-        <div className="login-container">
-          <Helmet>
-            <title>Login</title>
-          </Helmet>
+  return (
+    <div className="page">
+      <Helmet>
+        <title>Login</title>
+      </Helmet>
+      <div className="container">
+        <div>
+          <img src={WelcomeImg} className="WelcomeImg" alt="GSG Code Academy" />
+        </div>
+        <div className="right__content">
           <SimpleCard
             ClassName="card_sign"
             content={
-              <>
-                <Typography variant="h4" align="center" color="primary">
-                  Sign In
-                </Typography>
-                <Typography variant="h6" align="center" color="secondary">
-                  Please enter your login details
-                </Typography>
-                <div className="login-input-container">
+              <div>
+                <div className="card_sign__head">
+                  <Typography variant="h6" color="primary">
+                    Sign In
```

Comment:
To be similar to other routes & buttons names in all layouts.
```suggestion
                    Login
```